src/flask/blueprints.py

```diff
@@ -340,11 +340,10 @@ def extend(bp_dict, parent_dict):
                 app.cli.add_command(self.cli)
 
         for blueprint, bp_options in self._blueprints:
-            url_prefix = options.get("url_prefix", "")
-            if "url_prefix" in bp_options:
-                url_prefix = (
-                    url_prefix.rstrip("/") + "/" + bp_options["url_prefix"].lstrip("/")
-                )
+            url_prefix = options.get("url_prefix", self.url_prefix)
```

Comment:
I think this should be,
```python
url_prefix = options.get("url_prefix") or self.url_prefix                                                                                                                                       
bp_url_prefix = bp_options.get("url_prefix") or blueprint.url_prefix                                                                                                                            
if bp_url_prefix and url_prefix:                                                                                                                                                                
    url_prefix = url_prefix.rstrip("/") + "/" + bp_url_prefix.lstrip("/")
```
As it is possible for `url_prefix` to be in the options with a value of `None` and the net url_prefix to also be `None`. What do you think? (This is my [fix](https://gitlab.com/pgjones/quart/-/commit/29e4157583fd10bd504346fc3f319908339acc8d) for Quart).

Reply:
That makes sense, currently rebasing this PR, I'll make that change.